sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/SimplifyConditionalSuite.scala

```diff
@@ -166,4 +176,9 @@ class SimplifyConditionalSuite extends PlanTest with PredicateHelper {
         Literal(1))
     )
   }
+
+  test("simplify NOT(IsNull(x)) and NOT(IsNotNull(x))") {
+    assertFilter(Not(IsNotNull(UnresolvedAttribute("b"))), IsNull(UnresolvedAttribute("b")))
```

Comment:
We cannot use `assertEquivalent` here?

Reply:
> We cannot use `assertEquivalent` here?

`assertEquivalent` use OneRowRelation, no column, so I add a `testRelation`. 
I will remove it to `BooleanSimplificationSuit` .